.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: objective-c
+cache: cocoapods
```

Comment:
I am just poking around, since CocoaPods mysteriously failed to grab the latest master repo.